db/helpers/knex.js

```diff
@@ -0,0 +1,11 @@
+const path = require("path");
+const paths = require("./paths");
+
+const knex = require("knex")({
```

Comment:
This should be avoided. It also means that you setup a db connection (potentially) when a module is required.